app/models/db_client.py

```diff
@@ -46,6 +46,18 @@ def get_asset(id):
     return asset
 
 
+# TODO: move session creating and closing out of these individual calls
+def delete_asset(id):
+    try:
+        session = Session()
+        session.query(Asset).filter(Asset.id == id).delete(synchronize_session="fetch")
```

Comment:
Out of curiousity, what does `synchronize_session="fetch"` do?

Reply:
As per discussion offline:
![image](https://user-images.githubusercontent.com/13268990/109398385-afbd8880-790a-11eb-824c-25017ff6c5e5.png)

Will investigate if delete needs to be synchronized or not.